document/js-api/index.bs

```diff
@@ -325,74 +317,46 @@ A {{Module}} object represents a single WebAssembly module. Each {{Module}} obje
             1. If |v| is not a {{Table}} instance, throw a {{LinkError}} exception.
 
             Note: The table's length, etc. is checked by [=instantiate_module=] invoked below.
 
-            2. [=Append=] |v| to |tables|.
-            1. Let |tableaddr| be |v|.\[[Table]]
+            2. Let |tableaddr| be |v|.\[[Table]]
             1. Let |externtable| be the [=external value=] [=external value|𝗍𝖺𝖻𝗅𝖾=] |tableaddr|.
             1. [=Append=] |externtable| to |imports|.
-            1. For each element |func| of |v|.\[[Values]],
-                1. If |func| is not null, append |func| to |funcs|.
-    1. Let (|store|, |instance|) be [=instantiate_module=](|store|, |module|, |imports|), up through step 16 of [=the instantiation algorithm=], but omitting execution of the start function.
-    1. If |instance| is [=error=], throw a {{LinkError}} exception, or other <a href="#errors">appropriate exception type</a>.
-    1. For each |tableaddr| in |instance|.𝗍𝖺𝖻𝗅𝖾𝖺𝖽𝖽𝗋𝗌,
-        1. If there is no element in |tables| whose |table|.\[[Table]] is |tableaddr|:
-            1. Let |table| be [=create a table object|a new table object=] from |tableaddr|.
-            1. [=Append=] |table| to |tables|.
-        1. Otherwise:
-            1. Let |table| be the element in |tables| where |table|.\[[Table]] is |tableaddr|.
-        1. For each index |i| from 0 to the length of |table|.\[[Values]] &minus; 1, do
-            1. Let |funcaddr| be [=read_table=](|store|, |tableaddr|, |i|).
-            1. Assert: |funcaddr| is not [=error=] because this read is in bounds.
-            1. If |funcaddr| is not empty,
-                1. If |funcs| contains any |func| where |func|.\[[ExportedAddress]] is |funcaddr|,
-                    1. Let |func| be that function object.
-                1. Otherwise:
-                    1. Let |func| be a [=a new Exported Function=] created from |funcaddr|.
-                    1. [=Append=] |func| to |funcs|.
-                1. Set the |table|.\[[Values]][|i|] to |func|.
-
-                Note: The table and function objects created by the above steps are only observable for tables that are either imported or exported.
-
-    1. Let |startfuncaddr| be the start function of |instance|.
-    1. Let (|store|, |ret|) be [=invoke_func=](|store|, |startfuncaddr|, empty).
-    1. If |ret| is [=error=], throw an exception. This exception should be a WebAssembly {{RuntimeError}} exception, unless otherwise indicated by <a href="#errors">the WebAssembly error mapping</a>.
-    1. Assert: |ret| is empty.
+    1. Let (|store|, |instance|) be [=instantiate_module=](|store|, |module|, |imports|).
+    1. If |instance| is [=error=], throw an appropriate exception type:
+        * A {{LinkError}} exception for most cases which occur during linking.
+        * If the error came when running the start function, throw a {{RuntimeError}} for most errors which occur from WebAssembly, or the error object propagated from inner ECMAScript code.
```

Comment:
You can remove the "most" here, I think.

Reply:
Well, if it's a stack overflow or OOM from WebAssembly, it will be different. Probably there is a better way to word this, though.